go/vt/vtctl/reparentutil/util.go

```diff
@@ -21,6 +21,8 @@ import (
 	"sync"
 	"time"
 
+	"vitess.io/vitess/go/vt/vtctl/reparentutil/promotionrule"
```

Comment:
can you put this with the other "vitess.io" imports please?

Reply:
Fixed